src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowCrudSpec.groovy

```diff
@@ -1321,6 +1322,39 @@ class FlowCrudSpec extends HealthCheckSpecification {
         !error && flowHelperV2.deleteFlow(flow.flowId)
     }
 
+    @Tidy
+    def "Unable to update to a flow with incorrect maxLatency input - #title"() {
+        given: "A flow"
+        def (Switch srcSwitch, Switch dstSwitch) = topology.activeSwitches
+        def flow = flowHelperV2.randomFlow(srcSwitch, dstSwitch)
+        flowHelperV2.addFlow(flow)
+
+        when: "Try to update the flow "
+        def flowInfo = northboundV2.getFlow(flow.flowId)
+        flowInfo = flowInfo.tap {it.maxLatency = maxLatencyValue
+            it.maxLatencyTier2 = maxLatencyTier2}
+        northboundV2.updateFlow(flowInfo.flowId,
+                flowHelperV2.toRequest(flowInfo))
+
+        then: "Bad Request response is returned"
+        def error = thrown(HttpClientErrorException)
+        error.statusCode == HttpStatus.BAD_REQUEST
+        def errorDetails = error.responseBodyAsString.to(MessageError)
+        errorDetails.errorMessage == message
+        errorDetails.errorDescription == errorDescr
```

Comment:
Let's reuse approach from lines 598-599 of this file. If you can improve the one - it would be even better.

Reply:
done